joomla/joomla-cms
# Added plugin event triggers to the new mod_articles module

Pull Request for Issue # .

### Summary of Changes
With #43738 a new articles module is added. This new module mod_articles module has no option to trigger the plugin events. In the mod_articles_news this options was present. This PR adds the option to this module.

### Testing Instructions
1. Activate the option: Trigger Plugin Events.
2. Create three custom fields of type text and add then to each plugin event.
3. The custom field output is shown in for each article.

### Actual result BEFORE applying this Pull Request
No option to trigger plugin events.

### Expected result AFTER applying this Pull Request
Plugin events afterDisplayTitle, beforeDisplayContent and afterDisplayContent are triggered for each article.

### Link to documentations
Please select:
